src/context/AuthContext.js

```diff
@@ -0,0 +1,25 @@
+import { createContext } from "react";
+import { TOKEN } from "./../constants";
+
+const initialState = {
+  isAuthed: false,
+  authenticate: () => {},
+  logout: () => {},
+  user: null
+};
+
+const AuthContext = createContext(initialState);
```

Comment:
Move this into useAuth hook and don't export it, so that you're not tempted to import it in other places :)